examples/base-url/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  pyroscope:
+    image: 'pyroscope/pyroscope:latest'
+    ports:
+      - '4040:4040'
+    command:
+      - 'server'
+    environment:
+      PYROSCOPE_BASE_URL: /pyroscope
+  nginx:
+    image: nginx
+    network_mode: host
```

Comment:
this won't work on mac (and windows too I think)

it's more portable to refer to hosts using the hostnames:
```
proxy_pass http://pyroscope:4040;
```

and then you don't need `network_mode: host` either

Reply:
Yeah I was using that setup to test this base url stuff against a local pyroscope instance, and forgot to update that.